packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactClippingViewManager.kt

```diff
@@ -62,9 +62,6 @@ public abstract class ReactClippingViewManager<T : ReactViewGroup> : ViewGroupMa
     if (removeClippedSubviews) {
       val child = getChildAt(parent, index)
       if (child != null) {
-        if (child.parent != null) {
-          parent.removeView(child)
-        }
```

Comment:
Hmm, this is really interesting, and may explain a crash we're seeing.  I wonder why we were previously removing this subview twice - potentially messing up some state in ReactViewGroup.

What's your reasoning for removing this?